lib/elixir/lib/calendar.ex

```diff
@@ -420,6 +420,9 @@ defmodule Date do
   Converts the given `date` to
   [ISO 8601:2004](https://en.wikipedia.org/wiki/ISO_8601).
 
+  By default, Date.to_iso8601/2 returns dates formatted in the 'extended'
+  format, for human readability.  It also supports the 'basic' format through passing the :basic option.
```

Comment:
I believe "human readability" is the correct way here. http://www.grammarbook.com/punctuation/hyphens.asp